main.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=superfluous-parens
+import os
+from typing import Dict, Tuple
+from uuid import UUID
+
+import simplejson as json
+from flask import Request
+from notifications_python_client import __version__
+from notifications_python_client.authentication import create_jwt_token
+from requests import Session
+from requests.exceptions import RequestException
+
+session = Session()
+
+
+NOTIFY_API_KEY = os.environ["NOTIFY_API_KEY"]
+
+template_id_mapping = {
+    ("HH", "Eng", "en"): "0c5a4f95-bfa4-4364-9394-8499b4d777d5",
+    ("HH", "Wls", "en"): "0c5a4f95-bfa4-4364-9394-8499b4d777d5",
+    ("HH", "Wls", "cy"): "755d73d1-0cb6-4f2f-95e9-857a2ad071bb",
+    ("HH", "Nir", "en"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HH", "Nir", "ga"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HH", "Nir", "eo"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HI", "Eng", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "Wls", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "Wls", "cy"): "1001ac43-093d-425c-ac7d-68df5147c603",
+    ("HI", "Nir", "en"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "Nir", "ga"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "Nir", "eo"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("CE", "Eng", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "Wls", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "Wls", "cy"): "e4a4ebea-fcc8-463b-8686-5b8a7320f089",
+}
+
+
+# pylint: disable=too-many-return-statements
+def notify(request: Request) -> Tuple[str, int]:
```

Comment:
We should log our response. I can see the status code in GCP but would be useful to see the request which allows us to also log things like the `tx_id` with it.

Reply:
There is a print statement that covers this on L83